libvips/foreign/heif2vips.c

```diff
@@ -0,0 +1,1267 @@
+/* load heif images with libheif
```

Comment:
How about moving the definition of `heifload` back into `heifload.c`, and putting the C API wrappers for all these modules into `foreign.c`?

Reply:
Ah, yes, that is definitely better. I'm not sure why I didn't think of this in the first place. :sweat_smile:

Reply:
OK, done!